scripts/training/train.py

```diff
@@ -491,6 +491,22 @@ def main(
     top_p: float = 1.0,
     seed: Optional[int] = None,
 ):
+    if tf32 and not (
+        torch.cuda.is_available() and torch.cuda.get_device_capability()[0] >= 8
+    ):
```

Comment:
Is there any documentation to link in a comment here, as to why this check?

Reply:
Added link.